config/roles.yml

```diff
@@ -326,3 +326,17 @@ security_analytics_ack_alerts:
   reserved: true
   cluster_permissions:
     - 'cluster:admin/opensearch/securityanalytics/alerts/*'
+
+# Roles for Hello World extension, roles that extensions would like to add should be ultimately be defined elsewhere
+extension_hw_greet:
```

Comment:
Is it possible to parse this from the Hello World config? I know you have made changes in the SDK, I am not sure what would be required to read in these roles as part of the registration/installation process.

Reply:
Yes, that is ultimately where these should go! I wanted to open up a PR to show REST-layer authz in action with an example role. This [issue](https://github.com/opensearch-project/security/issues/2595) is about letting an extension pre-define roles and sourcing them into the security configuration.